thumbor/src/thumbor_/populate_bucket_expired.py

```diff
@@ -0,0 +1,50 @@
+import asyncio
+import os
+from datetime import datetime, timedelta
+
+import aioboto3
+import pytz
+
+config = {
+    "endpoint_url": os.environ.get("EP_URL"),
+    "aws_access_key_id": os.environ.get("KEY_ID"),
+    "aws_secret_access_key": os.environ.get("ACCESS_KEY"),
+    "region_name": "eu-west-2",
+}
+
+uk_zone = pytz.timezone("Europe/London")
+
+lifespan_days = [1, 30, 180, 365, 1095]
+
+
+async def upload_file(s3_client, bucket_name, key, lifespan):
+    current_time = datetime.now(uk_zone)
+
+    expiration_date = current_time - timedelta(days=lifespan)
+    metadata = {
```

Comment:
We aren't yet using object tags?

Reply:
Same as above, I think we can use the tags in the stress tests we will conduct in the upcoming weeks?